src/main/java/io/kokuwa/maven/helm/AbstractHelmMojo.java

```diff
@@ -211,12 +228,30 @@ void helm(String arguments, String errorMessage, String stdin) throws MojoExecut
 		if (StringUtils.isNotEmpty(repositoryCache)) {
 			command += " --repository-cache=" + repositoryCache;
 		}
+		if (StringUtils.isNotEmpty(namespace)) {
+			command += " --namespace=" + namespace;
+		}
+		if (StringUtils.isNotEmpty(kubeApiServer)) {
+			command += " --kube-apiserver=" + kubeApiServer;
+		}
+		if (StringUtils.isNotEmpty(kubeAsUser)) {
+			command += " --kube-as-user=" + kubeAsUser;
+		}
+		if (StringUtils.isNotEmpty(kubeAsGroup)) {
+			command += " --kube-as-group=" + kubeAsGroup;
+		}
+		if (StringUtils.isNotEmpty(kubeToken)) {
+			command += " --kube-token=" + kubeToken;
```

Comment:
Can you adjust [StripSensitiveDataLog](https://github.com/kokuwaio/helm-maven-plugin/blob/92f19af74d5715c73aa7dbd9941fc642f65ef976/src/main/java/io/kokuwa/maven/helm/StripSensitiveDataLog.java#L18)? Both, this code and K8sCluster code should use `=`.